_generator/templates/parameters.edge

```diff
@@ -2,6 +2,9 @@
 #### {{ schema.title }}{{-- {#{{ schemaId }}} --}}
 @end
 {{{ schema.description }}}
+@if(schema.oneOfRoot)
+The operation supports the following mutually exclusive parameters.
```

Comment:
This is a bit clearer to me because it mentions a set of parameters:
```suggestion
This operation accepts one of the following mutually exclusive parameter sets.
```

Reply:
Thanks! I will regenerate it in #784.